src/Symfony/Component/Security/Csrf/CsrfTokenManager.php

```diff
@@ -79,7 +79,7 @@ public function getToken(string $tokenId): CsrfToken
         return new CsrfToken($tokenId, $this->randomize($value));
     }
 
-    public function refreshToken(#[\SensitiveParameter] string $tokenId): CsrfToken
+    public function refreshToken(string $tokenId): CsrfToken
```

Comment:
@GromNaN I think this one was a mistake. It was not added to the interface and all other CSRF token ids. WDYT?